src/main/java/core/basesyntax/serviceimpl/ReaderServiceImpl.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.serviceimpl;
+
+import core.basesyntax.service.ReaderService;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReaderServiceImpl implements ReaderService<List<String>> {
```

Comment:
There is an unnecessary empty line at the beginning of the class implementation. Please remove it to adhere to the coding standards.